ktesting/example_test.go

```diff
@@ -98,3 +98,22 @@ func ExampleNewLogger() {
 	// E...] I failed err="failure" what="something" data={field:1}
 	// I...] Logged at level 5.
 }
+
+func ExampleV() {
+	var buffer ktesting.BufferTL
+	config := ktesting.NewConfig(ktesting.Verbosity(1))
+	logger := ktesting.NewLogger(&buffer, config)
+
+	logger.Info("initial verbosity", "v", config.Verbosity().String())
+	logger.V(2).Info("now you don't see me")
+	if err := config.Verbosity().Set("2"); err != nil {
+		logger.Error(err, "setting verbosity")
+	}
+	logger.V(2).Info("now you see me")
+
+	fmt.Print(headerRe.ReplaceAllString(buffer.String(), "${1}...] "))
```

Comment:
I think it would be nice to have an example here where it reduced the level back to the old one after increasing it to 2.

Reply:
```golang
if err := config.Verbosity().Set("1"); err != nil {
		logger.Error(err, "setting verbosity")
	}
	logger.V(2).Info("now you don't see me")
	logger.Info("restored verbosity", "v", config.Verbosity().String())
```
Something like above would be nice. Can you please check if that makes sense to add this example here?

Reply:
Makes sense.

I also included a corresponding example for the textlogger: https://github.com/kubernetes/klog/compare/a418587c5bcce0fa4eb1177eb989ef56571de93c..534dcfb54246a96e961e0ef7a43ccf3083ffcaeb